src/Surfnet/StepupRa/RaBundle/Controller/VettingController.php

```diff
@@ -44,6 +44,9 @@ class VettingController extends Controller
      * @Template
      * @param Request $request
      * @return array|Response
+     *
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity) Form validation and error handling
+     * @SuppressWarnings(PHPMD.NPathComplexity) Form validation and error handling
```

Comment:
There's also domain logic ;) Could you add a reference to the Pivotal ticket here?